lib/helpers/UiBaseLocators.ts

```diff
@@ -170,6 +180,23 @@ export class UiBaseLocators {
     await this.changeBtn.click();
   }
 
+  async enterAliasName(aliasName: string) {
+    // Unlocks alias
+    await this.aliasLockBtn.click();
+    await this.aliasNameTxt.clear();
+    await this.aliasNameTxt.fill(aliasName);
+  }
+
+  async updateIcon(iconName: string) {
+    await this.iconBtn.click({force: true});
+    await this.searchForTypeToFilterValue(iconName);
+    await this.iconBugBtn.click();
+  }
+
+  async enterDescriptionForPropertyEditorWithName(propertyEditorName: string, description: string) {
+    await this.page.locator('umb-media-type-workspace-view-edit-property').filter({hasText: propertyEditorName}).getByLabel('description').fill(description);
```

Comment:
locator('umb-media-type-workspace-view-edit-property') could be in the constructor.